Resources/config/imagine.xml

```diff
@@ -84,12 +84,12 @@
 
         <!--  Templating helpers and extensions -->
 
-        <service id="liip_imagine.twig.extension" class="Liip\ImagineBundle\Templating\ImagineExtension" public="false">
+        <service id="liip_imagine.templating.filter_extension" class="Liip\ImagineBundle\Templating\FilterExtension" public="false">
             <tag name="twig.extension" />
-            <argument type="service" id="liip_imagine.cache.manager" />
+            <argument type="service" id="liip_imagine.templating.filter_helper" />
```

Comment:
Why couple the twig helper to the templating helper? I realise the mostly have the same code, but making one dependent on the other feels off. If there is an abstraction here I would say that both the twig extension and the templating helper should depend on a shared service.